client/state/stats/emails/selectors.js

```diff
@@ -38,7 +45,7 @@ export function isRequestingAlltimeEmailStats( state, siteId, postId, statType )
 	return state.stats.emails
 		? get(
 				state.stats.emails.requests,
-				[ siteId, postId, 'alltime', statType, 'requesting' ],
+				[ ...getDataPath( siteId, postId, 'alltime', statType, null ), 'requesting' ],
```

Comment:
Maybe this is personal preference, but I like better to have the last parameter with a default value more than passing `null` values to functions. What do you think?

Reply:
I've changed it!